### ISSUE TITLE
Wikidata name locking blocks adding names in other languages

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#6789]
https://www.openstreetmap.org/edit?editor=id#map=21/41.86650/12.50386

![Selection_014](https://user-images.githubusercontent.com/899988/63497988-47941800-c4c5-11e9-98b9-97204a2b0028.png)

**Comment by eliotb** [comment]
It is possible to add another language by going to the "All tags" and clicking "+"  
Entering "name:" will give a list of suggestions.
Not as smooth as using the + on the name entry you show.

**Comment by matkoniecz** [comment]
I edited issue title to make it more clear

**Comment by tohaklim** [comment]
I thought this was by design? Or are you referring to the use case where a wikidata-locked entry lacks multilingual names and novice users should be allowed to add them?

**Comment by ferdinand0101** [comment]
This is probably the case
In my opinion the + in the name field should not be locked but the already existing name should stay locked

**Comment by quincylvania** [comment]
@bhousel Do you have thoughts on how this should work? I think protecting existing names but allowing new ones would be okay.

**Comment by bhousel** [comment]
> @bhousel Do you have thoughts on how this should work? I think protecting existing names but allowing new ones would be okay.

Yes I agree - It's probably ok to allow people to change the multilingual names..

**Comment by lukf** [comment]
Why don't we offer an "Upgrade" to automatically add name tags for every language in the wikidata item? The name-locking implies that names are automatically synchronised with the wikidata item.

**Comment by SilentSpike** [comment]
>Why don't we offer an "Upgrade" to automatically add name tags for every language in the wikidata item?

One problem with this is that wikidata labels aren't actually the same thing as the name of what the item represents, they're just identifying labels. The actual names of things should be stored in properties.

Another is that a single user is unlikely to be able to confirm that all languages are named accurately.

**Comment by matkoniecz** [comment]
Note that Wikidata items would require review due to quality issues.

And anyway Wikidata items are often burdened by incompatible licences (for example Wikidata items include data, for example names, added from Google maps - sometimes indirectly, via Wikipedia).

**Comment by Adamant36** [comment]
> Note that Wikidata items would require review due to quality issues.
I can attest to that. Names on Wikidata aren't stable at all as there's some edit warring that goes on with them. So I don't think it would be good to go with them directly.

**Comment by real68er** [comment]
I think these are outdated prejudices. Wikidata is absolutely comparable with OSM in terms of quality. The data is similarly difficult to edit. The quality control is on a comparable level. 

The best solution would be to take the name (also the multilingual variants) from Wikidata.

A compromise would be to take the name from Wikidata if no name is entered in OSM.

**Comment by 1ec5** [comment]
Copying anything from Wikidata is off-topic and belongs in a different issue. The original request was to be able to add the feature’s name tags directly using the Name field when the feature has been “locked” due to a Wikidata tag. That’s why this issue is still open.

The purpose of locking the Name field is to prevent accidental breakage or very casual vandalism of important features (however defined). It is not intended to encourage mappers to eschew OSM’s `name` tag in favor of Wikidata labels or ensure consistency between OSM and Wikidata.

**Comment by 1ec5** [comment]
This appears to be the relevant code:

https://github.com/openstreetmap/iD/blob/407360943ce5db9d98e857f6148c7876d789ff37/modules/ui/fields/localized.js#L199-L202

**Comment by HarelM** [comment]
Here's an example I'm not sure I fully understand the UX behind it. 
The following point has a wikidata link, the wikidata has a link to an article in English, Hebrew and Arabic.
The OSM element has a Hebrew name and a Russian name but doesn't have a name in English, 
As a user (not an advanced one who is familiar with the tags) I can't change the non-existing name as it is locked:

![Image](https://github.com/user-attachments/assets/3d9cb172-0a4a-4fab-987a-a2ca04c516b3)

I would expect only the Hebrew name to be locked as this is the only name that exists both in Wikidata and in OSM, the English shouldn't be locked as it does not exists in OSM and the Russian should not be locked as it exists only in OSM.

Am I misinterpreting this wikidata lock feature?

**Comment by 1ec5** [comment]
As some of the older comments indicate, the lock on the Multilingual Name fields was probably unintentional. Your screenshot additionally demonstrates that an empty field probably shouldn’t be locked either. https://github.com/openstreetmap/iD/issues/6789#issuecomment-1825954649 points to the code that would probably be changed to fix this issue.
